docs/installation.md

```diff
@@ -0,0 +1,56 @@
+## Using the boilerplate to create your project
+1. Clone this repository.
+```sh
+git clone https://github.com/ColoredCow/wordpress-init.git
+```
+2. Change remote URL to your project github URL using the following command.
+```sh
+git remote add origin https://github.com/user/repo.git
+```
+3. Change the theme name to your project theme name
```

Comment:
mention which folder

Reply:
Sure, I have made changes in the recent commit to provide a better explanation.